docs/en/observability/triage-threshold-breaches.asciidoc

```diff
@@ -0,0 +1,44 @@
+[[triage-threshold-breaches]]
+= Triage threshold breaches
+++++
+<titleabbrev>Threshold breaches</titleabbrev>
+++++
+
+Threshold breaches occur when an {observability} data type reaches or exceeds the threshold set in your <<custom-threshold-alert,custom threshold rule>>.
+For example, you might have a custom threshold rule that triggers an alert when the total number of log documents with a log level of `error` reaches 100.
+
+To triage issues quickly, go to the alert details page:
+
+. Go to **{observability}** → **Alerts**.
+. From the Alerts table, click the image:images/icons/boxesHorizontal.svg[More actions] icon next to the alert and select **View alert details**.
+
+The alert details page shows information about the alert, including when the alert was triggered,
+the duration of the alert, and the last status update.
+If there is a "group by" field specified in the rule, the page also includes the source.
+You can follow the links to navigate to the rule definition.
+
+Explore charts on the page to learn more about the threshold breach:
+
+[role="screenshot"]
+image::images/log-threshold-breach.png[Alert details for log threshold breach]
+
+* The page includes a chart for each condition specified in the rule.
+These charts help you understand when the breach occurred and its severity.
+* If your rule is intended to detect log threshold breaches
+(that is, it has a single condition that uses a count aggregation),
+you can run a log rate analysis, assuming you have the required license.
+Running a log rate analysis is useful for detecting significant dips or spikes in the number of logs.
```

Comment:
Do we have a document for log rate analysis component that maybe we can link here?

cc @jasonrhodes @walterra

Reply:
There is a topic in the [Kibana Guide](https://www.elastic.co/guide/en/kibana/master/xpack-ml-aiops.html) that talks about log rate analysis, but I didn't link to it because it does not seem generic enough. It tells user they can find log rate analysis under Machine Learning > AIOps Labs.

Should I go ahead and link to it anyhow? IMO we need to figure out a strategy for documenting these embeddable components now that they may appear in multiple contexts.

Reply:
I see, indeed it does not seem suitable in this case, I was mostly thinking about an explanation of how this component works or how to understand this component, not where to find it in Kibana.

Reply:
@maryam-saeidi Do you think we should document the behavior of the log rate analysis component in the Observability docs, too? We've done that for serverless because the feature is available under AIOps. But I worry about all the places where we might end up duplicating content if this feature gets embedded in multiple places. Perhaps it makes more sense to cover log rate analysis more generically in the Kibana topic so that it can apply to all the places in the UI where we might use this component. WDYT?

Reply:
Sounds like a good idea to me!

Reply:
I agree, it would be great if we could update the Log Rate Analysis page we have to include details where the feature is available in other places in Kibana like the alert details page. Pinging @szabosteve, maybe you have some thoughts on this?

Reply:
@walterra @szabosteve Does it make sense to have a document/section on a page about how this component works (without a reference to where it is used) and link it to the places where we are using this component? (So separating the information about what this component does and where to find it)

> it would be great if we could update the Log Rate Analysis page we have to include details where the feature is available in other places in Kibana like the alert details page.

Does it mean we would also like to have an overview of places where this component is used in one place? Does it mean we need to keep that list up to date?

Reply:
For now (to get this PR resolved), I will point to the Kibana documentation for more information about using the log rate analysis feature. @szabosteve Let me know if you want me to open an issue about making the log rate analysis info in the Kibana Guide more relevant for other UIs that embed this component. It's worth mentioning in the Kibana topic that the component appears in other UIs besides Kibana. I don't think you want to list out all the UIs, though, because that list would probably get stale over time.